src/olympia/search/filters.py

```diff
@@ -67,7 +67,7 @@ def get_value_from_reverse_dict(self):
         return self.get_value_from_object_from_reverse_dict()
 
 
-class AddonAppVersionFilterParam(AddonFilterParam):
+class AddonAppVersionFilterParam(AddonQueryParam):
```

Comment:
mhmm, naming… if it's inheriting from `AddonQueryParam` it should be a `AddonAppVersionQueryParam` imho. I know, that just blows up the diff, I'm fine doing that later.